### DISCUSSION TITLE
`Autofocus` `<SearchBox>` property on react-instant-search-hooks

### DISCUSSION THREAD

**Discussion by Mat-moran** [algolia/instantsearch#5328] [Q&A] [answered]
Hi,

The property `autofocus` is available in the `SearchBox` from instantSearch.js and reactInstantSearch.js but not for reactInstantSearchHooks. Is there any reason? or it will be implemented in the future?

Thanks

**Comment by dhayab** [comment]
Hi, thanks for your feedback! This looks like an omission during our initial implementation of the React InstantSearch Hooks widget.

It is definitely something that can be useful to users, so we'll add this option in the coming weeks. I'll let you know once it's available.

**Comment by sarahdayan** [comment, marked as answer]
Hey @Mat-moran,

You can pass an `autoFocus` prop to `<SearchBox>` from [v6.32.0](https://github.com/algolia/react-instantsearch/releases/tag/v6.32.0).

Thanks for letting us know!
